libexec/ramalama/ramalama-run-core

```diff
@@ -58,7 +58,7 @@ def main(args):
         container=False, dryrun=False, engine=None, podman_keep_groups=False,
         image='quay.io/ramalama/ramalama', runtime='llama.cpp',
```

Comment:
Shold this be hard coded? or runtime? or use_model_store?

Reply:
Thanks must fix this

Reply:
Changed to None, to be honest this field is just misleading more than anything, at this stage we are already in a container, so regardless of what's in this parameter, we are not spinning up a new container